bringup/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+# ###############################################################################
+# Set minimum required version of cmake, project name and compile options
+# ###############################################################################
+cmake_minimum_required(VERSION 3.5)
+project(c3pzero_bringup)
```

```suggestion
project(c3pzero_bringup NONE)
```
Because this project seemingly compiles no code, you can tell CMake to skip checking for any compilers since that's unnecessary.